File: ClientApp/src/app/model/comment.ts
export interface Comment {
  commentId: number;

  // 作者信息
  username: string;
  nickname: string;
  avatarUrl: string;

  content: string;
  likes: number;

  // LikeStatus==0: 未被Like或者Dislike;
  // LikeStatus==1: 被该用户Like;
  // LikeStatus==-1: 被该用户Dislike;
  likeStatus: number;

  // yyyy-mm-dd-hh-mm
  commentTime: string;
}

File: Interface/Comment.cs
namespace WebAngular.Interface
{
    public class Comment
    {
        public int ArticleId { get; set; }
        public int CommentId { get; set; }
        // 评论所引用的评论Id
        public int ReferenceId { get; set; }
        
        public string Username { get; set; }
        public string NickName { get; set; }
        public string AvatarUrl { get; set; }
        
        // yyyy-mm-dd-hh-mm
        public string CreateTime { get; set; }
        
        public string Content { get; set; }
    }
}